xdsl/dialects/builtin.py

```diff
@@ -910,6 +910,19 @@ def from_strings(name: str, value: str,
         return OpaqueAttr([StringAttr(name), StringAttr(value), type])
 
 
+@irdl_op_definition
+class UnrealizedConversionCastOp(Operation):
+    name: str = "builtin.unrealized_conversion_cast"
+
+    input: VarOperand
+    output: VarOpResult
+
+    @staticmethod
+    def get(input: SSAValue | Operation, result_type: Attribute):
```

Comment:
```suggestion
    def get(input: Sequence[SSAValue | Operation | Sequence[SSAValue | Operation]], result_types: Sequence[Attribute]):
```
Since it's a VarOpResult

Reply:
Maybe also a validation function that checks that the `len`s are the same?

Reply:
This raises Pyright issues, any idea how to fix?

Reply:
> Maybe also a validation function that checks that the `len`s are the same?

The lens of these are not at all the same actually, it allows 0 to n or n to 0 casts as well. I don't know why tho.